source/state/vault/index.ts

```diff
@@ -73,6 +75,15 @@ const VaultState = createSlice({
         [prefix]: value,
       };
     },
+    setEthereumTransactions(
+      state: IVaultState,
+      action: PayloadAction<TransactionResponse[] | any>
+    ) {
+      state.accounts[state.activeAccount.id] = {
+        ...state.accounts[state.activeAccount.id],
+        ethTransactions: action.payload,
```

Comment:
here we can use the existing property "transactions" for the active account, just adapting the types for each chain